internal/commands/image/rm.go

```diff
@@ -0,0 +1,56 @@
+package image
+
+import (
+	"fmt"
+
+	"github.com/docker/app/internal/store"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/distribution/reference"
+	"github.com/spf13/cobra"
+)
+
+func rmCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "rm [APP_IMAGE] [APP_IMAGE...]",
+		Short: "Remove an application image",
+		Args:  cli.RequiresMinArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			appstore, err := store.NewApplicationStore(config.Dir())
+			if err != nil {
+				return err
+			}
+
+			bundleStore, err := appstore.BundleStore()
+			if err != nil {
+				return err
+			}
+
+			for _, arg := range args {
+				ref, err := reference.ParseNormalizedNamed(arg)
+				if err != nil {
+					fmt.Println("Error:", err.Error())
```

Comment:
@thaJeztah WDYT? If an app-name is invalid, should we error out first or print it and continue?
I think this behavior exits with 0 even if a command line argument is invalid.
It may be easier to use, but I wonder if it fits with docker/cli ux?

Reply:
You are right, if one fails the error should not be 0, will fix